From: Philip Withnall <withnall@endlessm.com>
Date: Thu, 20 Sep 2018 17:51:18 +0100
Subject: tests: Mark two more tests as slow

These keep on taking just longer than 30s on my local machine when run
in parallel with the rest of the tests (i.e. with `ninja test`). Testing
them individually, they do terminate correctly.

Signed-off-by: Philip Withnall <withnall@endlessm.com>
Origin: upstream, 2.59.0, commit:18456b74a647907c8034418269b2cd33e68ffc5d
---
 gio/tests/meson.build      | 5 ++++-
 tests/refcount/meson.build | 2 +-
 2 files changed, 5 insertions(+), 2 deletions(-)

diff --git a/gio/tests/meson.build b/gio/tests/meson.build
index 4e5ad25..0dafef1 100644
--- a/gio/tests/meson.build
+++ b/gio/tests/meson.build
@@ -221,7 +221,10 @@ if host_machine.system() != 'windows'
         'extra_sources' : extra_sources,
         'suite' : ['slow'],
       },
-      'gmenumodel' : {'extra_sources' : extra_sources},
+      'gmenumodel' : {
+        'extra_sources' : extra_sources,
+        'suite' : ['slow'],
+      },
       'gnotification' : {
         'extra_sources' : [extra_sources, 'gnotification-server.c'],
       },
diff --git a/tests/refcount/meson.build b/tests/refcount/meson.build
index 9046bf7..afec051 100644
--- a/tests/refcount/meson.build
+++ b/tests/refcount/meson.build
@@ -1,5 +1,5 @@
 refcount_tests = [
- ['closures', 'closures.c', []],
+ ['closures', 'closures.c', [], 90],
  ['objects', 'objects.c', []],
  ['objects2', 'objects2.c', [], 90],
  ['properties', 'properties.c', []],
